probe-rs/src/bin/probe-rs/cmd/cargo_flash/diagnostics.rs

```diff
@@ -29,42 +29,6 @@ pub(crate) fn render_diagnostics(error: OperationError) {
                 "For a guide on how to set up your probes, see https://probe.rs/docs/getting-started/probe-setup".into(),
             ],
         ),
-        OperationError::FailedToOpenElf { source, path } => (
```

Comment:
I would like to keep those sections, even if they are not used currently. Anyone could use the error variant at any given time theoretically in which case the error info will be limited.